### DISCUSSION TITLE
When I import @web3modal/wagmi/vue, it keeps prompting me that it is not installed

### DISCUSSION THREAD

**Discussion by sutaiyi** [WalletConnect/walletconnect-monorepo#3578] [Developer Support]
When I import @web3modal/wagmi/vue, I keep reminding me that @web3modal/wagmi/vue is not installed. It is very strange that I can confirm that I have installed it. I can find the corresponding @web3modal/wagmi on node_modules, but there is no vue project underneath. , the version I installed "@web3modal/wagmi": "^3.0.0-alpha.4", the project environment is node v16.20.1, @vue/cli 5.0.8, "vue": "^3.0.0"; Can any expert tell me why?

**Comment by glitch-txs** [comment]
Hi can you try installing `npm i @web3modal/wagmi@3.0.0-alpha.5` ?

**Reply by sutaiyi** [reply]
Thank you for your answer, but I still can't solve it after replacing it, the problem still exists.

**Reply by glitch-txs** [reply]
hi, could you try adding `"type": "module",` to your package.json?
